### PR TITLE
change-tiup-related-content

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs/pull/6918#issuecomment-976214716

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by srstack — APPROVED** [review]

File: tiup/tiup-playground.md [hunk 1]
```diff
@@ -28,9 +28,11 @@ playground 的命令行参数说明：
 ```bash
 Flags:
       --db int                   设置集群中的 TiDB 数量（默认为1）
+      --db.host host             指定 TiDB 的监听地址
+      --db.port int              指定 TiDB 的端口号
       --db.binpath string        指定 TiDB 二进制文件的位置（开发调试用，可忽略）
       --db.config string         指定 TiDB 的配置文件（开发调试用，可忽略）
-      --db.host host             指定 TiDB 的监听地址
+      --db.timeout int           指定 TiDB 最大等待超时时间，单位为秒，若配置为0，则为用不超时。
```
Comment (TomShawn) [inline tiup/tiup-playground.md:35]:
```suggestion
      --db.timeout int           指定 TiDB 最长等待超时时间，单位为秒。若配置为 0，则永不超时。
```

File: tiup/tiup-playground.md [hunk 2]
```diff
@@ -39,16 +41,25 @@ Flags:
       --kv int                   设置集群中的 TiKV 数量（默认为1）
       --kv.binpath string        指定 TiKV 二进制文件的位置（开发调试用，可忽略）
       --kv.config string         指定 TiKV 的配置文件（开发调试用，可忽略）
-      --monitor                  是否启动监控
+      --mode string              指定 TiUP playground 运行模式，支持'tidb', 'tikv-slim' (默认为 "tidb")
```
Comment (TomShawn) [inline tiup/tiup-playground.md:44]:
```suggestion
      --mode string              指定 TiUP playground 运行模式，可选值为 'tidb'（默认）和 'tikv-slim'
```

File: tiup/tiup-playground.md [hunk 3]
```diff
@@ -39,16 +41,25 @@ Flags:
...
       --pd int                   设置集群中的 PD 数量（默认为1）
+      --pd.host host             指定 PD 的监听地址
       --pd.binpath string        指定 PD 二进制文件的位置（开发调试用，可忽略）
       --pd.config string         指定 PD 的配置文件（开发调试用，可忽略）
       --pump int                 指定集群中 Pump 的数量（非 0 的时候 TiDB 会开启 TiDB Binlog）
       --pump.binpath string      指定 Pump 二进制文件的位置（开发调试用，可忽略）
       --pump.config string       指定 Pump 的配置文件（开发调试用，可忽略）
+  -T, --tag string               设置 playground 的 tag 信息
```
Comment (TomShawn) [inline tiup/tiup-playground.md:52]:
```suggestion
  -T, --tag string               设置 TiUP playground 的 tag 信息
```

File: tiup/tiup-playground.md [hunk 4]
```diff
@@ -39,16 +41,25 @@ Flags:
...
+      --ticdc int                设置集群中的 TiCDC 的数量（默认为0）
```
Comment (TomShawn) [inline tiup/tiup-playground.md:53]:
```suggestion
      --ticdc int                设置集群中的 TiCDC 节点的数量（默认为 0）
```

File: tiup/tiup-playground.md [hunk 5]
```diff
@@ -39,16 +41,25 @@ Flags:
...
+      --ticdc.binpath string     指定 TiCDC 二进制文件的位置（开发调试用，可忽略）
+      --ticdc.config string      指定 TiCDC 的配置文件（开发调试用，可忽略）
       --tiflash int              设置集群中 TiFlash 数量（默认为0）
       --tiflash.binpath string   指定 TiFlash 的二进制文件位置（开发调试用，可忽略）
       --tiflash.config string    指定 TiFlash 的配置文件（开发调试用，可忽略）
+      --tiflash.timeout int      指定 TiFlash 最大等待超时时间，单位为秒，若配置为0，则为用不超时。
```
Comment (TomShawn) [inline tiup/tiup-playground.md:59]: 同上

File: tiup/tiup-playground.md [hunk 6]
```diff
@@ -39,16 +41,25 @@ Flags:
...
+  -v, --version                  显示 TiUP playground 的版本号
+      --without-monitor          设置不使用 prometheus 和 grafana 的监控功能 ,若无此参数则默认开启监控功能。
```
Comment (TomShawn) [inline tiup/tiup-playground.md:61]:
```suggestion
      --without-monitor          设置不使用 Prometheus 和 Grafana 的监控功能。若不添加此参数则默认开启监控功能。
```

**Comment by zhangyangyu** [comment]
One more thing to notice is that in `tiup/tiup-playground.md`, it says:

> 如果直接执行 tiup playground 命令，则 TiUP playground 会使用本地安装的 TiDB/TiKV/PD 组件或者安装这些组件的稳定版本，来启动一个由 1 个 TiKV、1 个 TiDB、1 个 PD 实例构成的集群。该命令实际做了以下事情：

I think currently it will also start a tiflash instance. Not sure, can @srstack confirm it?

**Comment by srstack** [comment]
yes, tiup playground will start a tiflash instance by default mode.

**Review by AstroProfundis — APPROVED** [review]
Contents beside current comments are LGTM.

File: tiup/tiup-playground.md [hunk 7]
```diff
@@ -39,16 +41,25 @@ Flags:
...
       --pd.binpath string        指定 PD 二进制文件的位置（开发调试用，可忽略）
       --pd.config string         指定 PD 的配置文件（开发调试用，可忽略）
       --pump int                 指定集群中 Pump 的数量（非 0 的时候 TiDB 会开启 TiDB Binlog）
       --pump.binpath string      指定 Pump 二进制文件的位置（开发调试用，可忽略）
       --pump.config string       指定 Pump 的配置文件（开发调试用，可忽略）
+  -T, --tag string               设置 TiUP playground 的 tag 信息
+      --ticdc int                设置集群中的 TiCDC 节点的数量（默认为 0）
+      --ticdc.binpath string     指定 TiCDC 二进制文件的位置（开发调试用，可忽略）
+      --ticdc.config string      指定 TiCDC 的配置文件（开发调试用，可忽略）
       --tiflash int              设置集群中 TiFlash 数量（默认为0）
```
Comment (qqqdan) [inline tiup/tiup-playground.md:56]:
```suggestion
      --tiflash int              设置集群中 TiFlash 节点的数量（默认为 1）
```

File: tiup/tiup-playground.md [hunk 8]
```diff
@@ -39,16 +41,25 @@ Flags:
...
       --pump.config string       指定 Pump 的配置文件（开发调试用，可忽略）
-      --tiflash int              设置集群中 TiFlash 数量（默认为0）
+  -T, --tag string               设置 TiUP playground 的 tag 信息
+      --ticdc int                设置集群中的 TiCDC 节点的数量（默认为 0）
+      --ticdc.binpath string     指定 TiCDC 二进制文件的位置（开发调试用，可忽略）
+      --ticdc.config string      指定 TiCDC 的配置文件（开发调试用，可忽略）
+      --tiflash int              设置集群中 TiFlash 节点的数量（默认为 1）
       --tiflash.binpath string   指定 TiFlash 的二进制文件位置（开发调试用，可忽略）
       --tiflash.config string    指定 TiFlash 的配置文件（开发调试用，可忽略）
+      --tiflash.timeout int      指定 TiFlash 最长等待超时时间，单位为秒，若配置为 0，则为用不超时。
```
Comment (shichun-0415) [inline tiup/tiup-playground.md:59]:
```suggestion
      --tiflash.timeout int      指定 TiFlash 最长等待超时时间，单位为秒，若配置为 0，则为永不超时。
```

File: tiup/tiup-playground.md [hunk 9]
```diff
@@ -39,16 +41,25 @@ Flags:
...
+  -v, --version                  显示 TiUP playground 的版本号
+      --without-monitor          设置不使用 Prometheus 和 Grafana 的监控功能。若不添加此参数则默认开启监控功能。
```
Comment (shichun-0415) [inline tiup/tiup-playground.md:61]:
```suggestion
      --without-monitor          设置不使用 Prometheus 和 Grafana 的监控功能。若不添加此参数，则默认开启监控功能。
```

File: tiup/tiup-playground.md [hunk 10]
```diff
@@ -28,9 +28,11 @@ playground 的命令行参数说明：
 ```bash
 Flags:
       --db int                   设置集群中的 TiDB 数量（默认为1）
```
Comment (shichun-0415) [inline tiup/tiup-playground.md:30]:
```suggestion
      --db int                   设置集群中 TiDB 节点的数量（默认为1）
```

File: tiup/tiup-playground.md [hunk 11]
```diff
@@ -39,16 +41,25 @@ Flags:
       --kv int                   设置集群中的 TiKV 数量（默认为1）
```
Comment (shichun-0415) [inline tiup/tiup-playground.md:41]:
```suggestion
      --kv int                   设置集群中 TiKV 节点的数量（默认为1）
```

File: tiup/tiup-playground.md [hunk 12]
```diff
@@ -39,16 +41,25 @@ Flags:
...
       --kv.binpath string        指定 TiKV 二进制文件的位置（开发调试用，可忽略）
       --kv.config string         指定 TiKV 的配置文件（开发调试用，可忽略）
-      --monitor                  是否启动监控
+      --mode string              指定 TiUP playground 的运行模式，取值选项为 'tidb'（默认）和 'tikv-slim'
       --pd int                   设置集群中的 PD 数量（默认为1）
```
Comment (shichun-0415) [inline tiup/tiup-playground.md:45]:
```suggestion
      --pd int                   设置集群中 PD 节点的数量（默认为1）
```

File: tiup/tiup-playground.md [hunk 13]
```diff
@@ -39,16 +41,25 @@ Flags:
...
+      --pd.host host             指定 PD 的监听地址
       --pd.binpath string        指定 PD 二进制文件的位置（开发调试用，可忽略）
       --pd.config string         指定 PD 的配置文件（开发调试用，可忽略）
       --pump int                 指定集群中 Pump 的数量（非 0 的时候 TiDB 会开启 TiDB Binlog）
```
Comment (shichun-0415) [inline tiup/tiup-playground.md:49]:
```suggestion
      --pump int                 设置集群中 Pump 节点的数量（非 0 的时候 TiDB 会开启 TiDB Binlog）
```

File: tiup/tiup-playground.md [hunk 14]
```diff
@@ -39,16 +41,25 @@ Flags:
...
       --pump.binpath string      指定 Pump 二进制文件的位置（开发调试用，可忽略）
       --pump.config string       指定 Pump 的配置文件（开发调试用，可忽略）
-      --tiflash int              设置集群中 TiFlash 数量（默认为0）
+  -T, --tag string               设置 TiUP playground 的 tag 信息
+      --ticdc int                设置集群中的 TiCDC 节点的数量（默认为 0）
```
Comment (shichun-0415) [inline tiup/tiup-playground.md:53]:
```suggestion
      --ticdc int                设置集群中 TiCDC 节点的数量（默认为 0）
```

**Review by shichun-0415 — COMMENTED** [review]
The rest looks good to me.

File: tiup/tiup-playground.md [hunk 15]
```diff
@@ -15,40 +15,51 @@ playground 组件的基本用法：
...
       --drainer.config string    指定 Drainer 的配置文件
   -h, --help                     打印帮助信息
       --host string              设置每个组件的监听地址（默认为 127.0.0.1），如果要提供给别的电脑访问，可设置为 0.0.0.0
-      --kv int                   设置集群中的 TiKV 数量（默认为1）
+      --kv int                   设置集群中 TiKV 节点的数量（默认为1）
       --kv.binpath string        指定 TiKV 二进制文件的位置（开发调试用，可忽略）
       --kv.config string         指定 TiKV 的配置文件（开发调试用，可忽略）
-      --monitor                  是否启动监控
-      --pd int                   设置集群中的 PD 数量（默认为1）
+      --mode string              指定 TiUP playground 的运行模式，取值选项为 'tidb'（默认）和 'tikv-slim'
```
Comment (qqqdan) [inline tiup/tiup-playground.md:44]:
```suggestion
      --mode string              指定 playground 的运行模式，取值选项为 'tidb'（默认）和 'tikv-slim'
```

File: tiup/tiup-playground.md [hunk 16]
```diff
@@ -15,40 +15,51 @@ playground 组件的基本用法：
...
+      --pd int                   设置集群中 PD 节点的数量（默认为1）
+      --pd.host host             指定 PD 的监听地址
       --pd.binpath string        指定 PD 二进制文件的位置（开发调试用，可忽略）
       --pd.config string         指定 PD 的配置文件（开发调试用，可忽略）
-      --pump int                 指定集群中 Pump 的数量（非 0 的时候 TiDB 会开启 TiDB Binlog）
+      --pump int                 设置集群中 Pump 节点的数量（非 0 的时候 TiDB 会开启 TiDB Binlog）
       --pump.binpath string      指定 Pump 二进制文件的位置（开发调试用，可忽略）
       --pump.config string       指定 Pump 的配置文件（开发调试用，可忽略）
-      --tiflash int              设置集群中 TiFlash 数量（默认为0）
+  -T, --tag string               设置 TiUP playground 的 tag 信息
```
Comment (qqqdan) [inline tiup/tiup-playground.md:52]:
```suggestion
  -T, --tag string               设置 playground 的 tag 信息
```

File: tiup/tiup-playground.md [hunk 17]
```diff
@@ -15,40 +15,51 @@ playground 组件的基本用法：
...
+      --ticdc int                设置集群中 TiCDC 节点的数量（默认为 0）
+      --ticdc.binpath string     指定 TiCDC 二进制文件的位置（开发调试用，可忽略）
+      --ticdc.config string      指定 TiCDC 的配置文件（开发调试用，可忽略）
+      --tiflash int              设置集群中 TiFlash 节点的数量（默认为 1）
       --tiflash.binpath string   指定 TiFlash 的二进制文件位置（开发调试用，可忽略）
       --tiflash.config string    指定 TiFlash 的配置文件（开发调试用，可忽略）
+      --tiflash.timeout int      指定 TiFlash 最长等待超时时间，单位为秒，若配置为 0，则永不超时。
+  -v, --version                  显示 TiUP playground 的版本号
```
Comment (qqqdan) [inline tiup/tiup-playground.md:60]:
```suggestion
  -v, --version                  显示 playground 的版本号
```

**Review by zhangyangyu — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**